web/src/app/alerts/components/AlertDetailLogs.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import p from 'prop-types'
+import Divider from '@material-ui/core/Divider'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Button from '@material-ui/core/Button'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+import { DateTime, Interval } from 'luxon'
+import { POLL_INTERVAL } from '../../config'
+
+const LIMIT = 149
+
+const query = gql`
+  query getAlert($id: Int!, $input: AlertRecentEventsOptions) {
+    alert(id: $id) {
+      data: recentEvents(input: $input) {
```

Comment:
It would probably make more sense to just have this be `recentEvents` without the `data` alias, so that the `_.get` variables are more understandable later on

e.g.
`const events = _.get(data, 'alert.data.nodes', [])`
->
`const events = _.get(data, 'alert.recentEvents.nodes', [])`